WormholeConnect-Migration-Guide-v2.5.0-to-v3.0.0.md

```diff
@@ -0,0 +1,227 @@
+# WormholeConnect Migration Guide: v2.5.0 to v3.0.0
```

Comment:
https://wormhole.com/docs/products/connect/guides/upgrade/
We have a migration guide to get to 1.0.0, but we don't have anything from 1 to 2, and all of this should be in there to go from 2 to 3.